R/conditions.R

```diff
@@ -536,12 +536,12 @@ loss_type <- function(x) {
 # Used in maybe_warn_deprecated_lossy_cast()
 new_error_cast_lossy <- function(x, to, loss_type, x_arg = "", to_arg = "") {
   error_cnd(
-    "vctrs_error_cast_lossy",
+    c("vctrs_error_cast_lossy", "vctrs_error_incompatible_type"),
     x = x,
-    to = to,
+    y = to,
     loss_type = loss_type,
     x_arg = x_arg,
-    to_arg = to_arg
+    y_arg = to_arg
   )
 }
```

Comment:
It seems like it may be worth it to have `new_error_incompatible_type()` that this could use? To try to prevent them from ever getting out of sync?

Reply:
yes but that requires too many changes in conditions.R